core-modules.nix

```diff
@@ -65,7 +67,7 @@
                     in
                     [
                       {
-                        url = baseUrl + subpath;
+                        ${if baseUrl == null then null else "url"} = baseUrl + subpath;
                         name = sourceName + subpath;
```

Comment:
This is to support the case of no `rev` during development.

It seems the result of this is just fine. Without a `url` there simply is no link, but the file path is printed the same.